restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCDevice.java

```diff
@@ -370,6 +370,10 @@ public RCConnection connect(Map<String, Object> parameters, RCConnectionListener
             connection.state = RCConnection.ConnectionState.PENDING;
             DeviceImpl.GetInstance().sipuaConnectionListener = connection;
 
+            if (enableVideo == null) {
```

Comment:
@mehturt: Above where the assignment is made we are casting to Boolean so it should have the same effect. Did you have any issues with that?